makefile

```diff
@@ -109,6 +114,12 @@ wp8 ?= $(root)/wp8
 
 classpath = avian
 
+bootimage-classpath = $(classpath-build)
+
+ifeq ($(bootimage-test),true)
```

Comment:
To be consistent with the adjustment of `options` above, we might want to wrap this in a `ifeq ($(bootimage),true)`.  Otherwise, you could accidentally build with bootimage-test, but not bootimage and get unexpected results.

Better yet, maybe add a check to the parameter validation code.